Kentico/cloud-sample-app-react
# Add prettierrc file and update list-staged script

### Motivation





After pulling down the repo and starting to work on an issue, I turned on VS Code's `formatOnSave` option and saw that prettier was using double quotes, and the `lint-staged` script was using single quotes. To fix this, I created the `.prettierrc` file and removed the `--single-quotes` flag from the `lint-staged` command since running the prettier-cli resolves the config. This change will allow editors to use the same config.

### Checklist


### How to test



1. Switch some quotes to double quotes then commit. The `lint-staged` commit hook should convert them back to single quotes.
2. Try turning on `formatOnSave` with Prettier installed, and it should use single quotes based on the .prettierrc file
